docs/BR.md

```diff
@@ -1142,10 +1142,11 @@ The validity interval of an OCSP response is the difference in time between the
 
 For the status of Subscriber Certificates:
 
-1. OCSP responses MUST have a validity interval greater than or equal to eight hours;
-2. OCSP responses MUST have a validity interval less than seven days;
-3. For OCSP responses whose validity interval is less than or equal to sixteen hours, the CA SHALL update the information provided via an Online Certificate Status Protocol at least eight hours prior to the nextUpdate;
-4. For OCSP responses whose validity interval is greater than sixteen hours, the CA SHALL update the information provided via an Online Certificate Status Protocol prior to one-half of the validity period remaining before the nextUpdate.
+1.  OCSP responses for certificates must meet the following requirements:
+	a. Minimum validity of eight (8) hours; maximum validity of ten (10) days for leaf certificates and 12 months for subordinate CA certificates; and
+	b. If the validity period of the OCSP response is less than 16 hours, then the next response must be available at ½ the validity period.  If the validity period is greater than 16 hours then the next response must be available no less than 8 hours before the current response expires.
+2.  All subordinate CA certificates must include the CRL distribution point extension and an AIA containing an OCSP responder URL.  All end-entity certificates must include the AIA containing an OCSP responder URL and can include the CRL distribution point extension.
```

Comment:
```suggestion
```

So, this second bullet is more about the certificate profile, and is captured at https://github.com/sleevi/cabforum-docs/blame/2019-10-Browser_Alignment/docs/BR.md#L1589 (that is, Section 7.1.2.2(c), which is modified - https://github.com/sleevi/cabforum-docs/blob/2019-10-Browser_Alignment/docs/BR.md#7122-subordinate-ca-certificate ) . 7.1.2.2 (b) already requires CRLs for Subordinates. 

For Subscriber certificates, that's https://github.com/sleevi/cabforum-docs/blame/2019-10-Browser_Alignment/docs/BR.md#L1648 ( specifically, 7.1.2.3 (c))

https://github.com/sleevi/cabforum-docs/commit/c8d5a3ddb3385c587f3a203be81b03e8de9a959e was the commit that added these, so I think we can just delete this line item from this section.